### PR TITLE
Adding tic tac toe example

### PR DESCRIPTION
**Pull request by renanccastro**

### CODE REVIEW

File: tic-tac-toe/README.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+#Tic tac toe
```
Comment (filipenevola) [inline tic-tac-toe/README.md:1]: It's missing how to set up and run the example.

File: tic-tac-toe/package.json [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  "name": "tic-tac-toe",
+  "private": true,
+  "scripts": {
+    "start": "meteor run",
+    "test": "meteor test --once --driver-package meteortesting:mocha",
+    "test-app": "TEST_WATCH=1 meteor test --full-app --driver-package meteortesting:mocha",
+    "visualize": "meteor --production --extra-packages bundle-visualizer"
+  },
+  "dependencies": {
+    "@babel/runtime": "^7.11.2",
```
Comment (filipenevola) [inline tic-tac-toe/package.json:11]: could you update the dependencies?

File: tic-tac-toe/tests/main.js [hunk 3]
```diff
@@ -0,0 +1,20 @@
+import assert from "assert";
+
+describe("tic-tac-toe", function () {
```
Comment (filipenevola) [inline tic-tac-toe/tests/main.js:3]: could you remove it?

**Review by filipenevola — CHANGES_REQUESTED** [review]
Hi, thank you! :)

I've requested some changes and also you need to add your example to the main README

**Review by filipenevola — CHANGES_REQUESTED** [review]
I believe the application is not deployed at https://tic-tac-toe.meteorapp.com

Is it?

**Comment by renanccastro** [comment]
Holy moly, yes, it wasn't.
Now it is!

**Review by filipenevola — APPROVED** [review]
We could improve the UI before promoting it. What do you think?

**Pull request merged**